wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/OAuthUseCase.swift

```diff
@@ -42,7 +42,7 @@ public class OAuthUseCase: OAuthUseCaseInterface {
         logger.info("invoke authentication flow")
 
         guard let bundleID = Bundle.main.bundleIdentifier,
-              let redirectURI = URL(string: "\(bundleID):/oauth2redirect")
+              let redirectURI = URL(string: "wire://e2ei/oauth2redirect")
```

Comment:
Can it happen that users have multiple wire apps installed and then another app is called? 🤓

Reply:
`ASWebAuthenticationSession ensures that only the calling app’s session receives the authentication callback, even when more than one app registers the same callback URL scheme.`

https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession